src/displayBackend/drm/Display.cpp

```diff
@@ -244,6 +250,96 @@ FrameBufferPtr Display::createFrameBuffer(DisplayBufferPtr displayBuffer,
  * Private
  ******************************************************************************/
 
+bool Display::isDrmDevice(udev_device *device)
+{
+	const char *filename = udev_device_get_devnode(device);
+	const char *sysnum = udev_device_get_sysnum(device);
+	drmModeRes *res;
+	int id, fd;
+	bool ret = false;
+
+	if (!filename)
+		return false;
+
+	fd = open(filename, O_RDWR | O_CLOEXEC);
```

Comment:
I am wondering, do you need write access here?

Reply:
we just open it as [libdrm does](https://cgit.freedesktop.org/mesa/drm/tree/xf86drm.c#n412)